sdcm/cluster.py

```diff
@@ -458,6 +458,16 @@ def _init_port_mapping(self):
     def region(self):
         raise NotImplementedError()
 
+    def refresh_ip_address(self):
+        self.ssh_login_info["hostname"] = self.external_address
+        self.remoter.hostname = self.external_address
+
+        if (IP_SSH_CONNECTIONS == 'public' or Setup.MULTI_REGION) and Setup.RSYSLOG_ADDRESS:
+            ContainerManager.destroy_container(self, "auto_ssh:rsyslog", ignore_keepalive=True)
```

Comment:
It's better to move the code to `_init_port_mapping` and destroy this container without any conditions (just catch NotFound exception):

```python
try:
    ContainerManager.destroy_container(...)
except NotFound:
    pass
```

Reply:
(need to import `NotFound` from `sdcm.utils.docker_utils`)